### ISSUE TITLE
Execute horst without display

### ISSUE THREAD

**Issue by Sawyer13** [br101/horst#100]
Is there any options to execute horst without "display panel"? My idea is parser the information for use horst as submodule in a program that i'm building.

**Comment by br101** [comment]
I think the best option for this is to use Libuwufi directly...

**Comment by dlenski** [comment]
Yes, you can use `horst -q -o /dev/stdout` as a subprocess, and parse its CSV output. I've been writing a Python module to do just that, actually.
